xcell/tests/test_cls.py

```diff
@@ -294,6 +298,13 @@ def test_symmetric():
     shutil.rmtree(tmpdir1)
 
 
+def test_unsupported_quantity():
+    data = get_config(dtype0='generic')
```

Comment:
`generic` is supported now. Should you use a different one (like `MyVeryUnsupportedTracerType`)?

Reply:
`generic` is not supported for `ClFid`. And if I changed it to anything else, an error will be thrown because `MapperDummy` does not support this type (and I want to check the `ClFid` error instead)

Reply:
Understood.